package/boot/uboot-lantiq/Makefile

```diff
@@ -229,6 +229,25 @@ define U-Boot/easy50712_norspl
   DEPENDS+=@BROKEN
 endef
 
+define U-Boot/arv7519rw22_brn
```

Comment:
The install instructions at https://openwrt.org/toh/arcadyan/arv7519rw22#installation explain how to use BRNBoot to flash the NOR version of U-boot. The BRN version of U-boot is most likely not needed. Has it been tested?